### PR TITLE
Checkout page

### PR DESCRIPTION
**Pull request by ZachOBrien**
image

Open to suggestions on handling swapping between pagination and scroll differently. Right now the "Events" component takes a boolean prop to use pagination if true, seeing as we use scroll much more frequently.

### CODE REVIEW

File: src/components/Checkout/Checkout.vue [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    <h1>My Cart</h1>
+    <h3>You signed up for the following events:</h3>
+    <div class="component-container">
+      <div class="events component-wrapper">
+        <events />
+      </div>
+      <div class="payment component-wrapper"></div>
+      <div class="codes component-wrapper"></div>
+    </div>
+    <the-footer />
```
Comment (bensenescu) [inline src/components/Checkout/Checkout.vue]: the-footer should be moved to app.vue if its not already there.

File: assets/color-constants.less [hunk 2]
```diff
@@ -1,7 +1,10 @@
...
+@header-color: #3da848;
 @body-color: white;
 @notes-form-color: #c4d2ff;
 
 @button-bg: black;
 @button-color: #eeeeee;
 @button-hover-color: white;
+
+@green-apple: #5FBD5C;
+@tangerine: #FBA34A;
```
Comment (bensenescu) [inline assets/color-constants.less]: thanks for adding those

File: src/components/Events/Events.vue [hunk 3]
```diff
@@ -1,22 +1,38 @@
 <template>
-  <div>
-    <h1>Events</h1>
-    <EventsList :events="events"></EventsList>
+  <div class="events-container">
+    <h1 v-if="isPaginate">Events</h1>
+    <events-list v-if="isPaginate" :events="events"></events-list>
+    <events-list-scroll v-else :events="events"></events-list-scroll>
```
Comment (bensenescu) [inline src/components/Events/Events.vue]: nice job

File: src/components/Events/Events.vue [hunk 4]
```diff
@@ -1,22 +1,38 @@
...
+    EventsListScroll,
+  },
+  props: {
+    paginate: { type: Boolean, default: false },
   },
   computed: {
     events() {
       return this.$store.getters['events/getEvents'];
     },
+    isPaginate() {
```
Comment (bensenescu) [inline src/components/Events/Events.vue]: No need for this computed property. Just use the prop.

File: src/components/Events/EventsList.vue [hunk 5]
```diff
@@ -24,6 +14,16 @@
             class='pagination__btn'
             v-on:click='incrementCurrentPage'>Next</button>
         </div>
+        <div v-if="!noEvents" class="events-container">
```
Comment (bensenescu) [inline src/components/Events/EventsList.vue]: why did you change this from the v-else?

File: assets/color-constants.less (hunk 2 above)
Comment (ZachOBrien) [reply assets/color-constants.less]: :)

File: src/components/Checkout/Checkout.vue (hunk 1 above)
Comment (ZachOBrien) [reply src/components/Checkout/Checkout.vue]: gotcha

File: src/components/Events/EventsList.vue (hunk 5 above)
Comment (ZachOBrien) [reply src/components/Events/EventsList.vue]: Bill could originally make this v-else because he swapped around where the pagination buttons were. He took them from the top of the component and put them on the bottom, meaning now the "events-container" came directly after the span with v-if, so v-else could work. But this isn't how it's laid out on figma. On figma the buttons are at the top of the list of events, and so the div that contains those buttons separates the "events-container" div from the span with v-if, and v-else no longer works.

**Comment by ZachOBrien** [comment]
> In the screenshot it looks like there's a big gap between the event title and its description. Is this intentional or should the vertical gap be smaller?

Good catch. It was a mistake on my part. I just made the adjustment.

File: src/components/Checkout/Checkout.vue (hunk 1 above)
Comment (ZachOBrien) [reply src/components/Checkout/Checkout.vue]: yep, will do
